resources/htmlTemplate/TocHTMLTemplate.html

```diff
@@ -136,21 +136,19 @@ <h1><a href="/" title="Home" rel="home">Software Package Data Exchange (SPDX)</a
-      URL for each license and exception.   
+      <p>The SPDX License List is an integral part of the SPDX Specification. The SPDX License List itself is a list of commonly found licenses and exceptions used in free and open or collaborative software, data, hardware, or documentation. The SPDX License List includes a standardized short identifier, the full name, the license text, and a canonical permanent URL for each license and exception.    
       </p>
-
+	<p>The purpose of the SPDX License List is to enable efficient and reliable identification of such licenses and exceptions in an SPDX document, in source files or elsewhere. </p>
+	    
       <ul style="list-style-type: square; padding: 5px;">
-            <li style="margin:5px 0 5px 10px;"><a href="./exceptions-index.html"><strong>License Exceptions</strong></a>: The list of commonly found exceptions to free and open source licenses, which can be used with the <strong><a href="https://spdx.org/specifications">License Expression</a></strong> operator, "WITH" to create a license with an exception. </li>
-            <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML"><strong>Master Files</strong></a>:  The HTML pages you see here are generated from the master files for the SPDX License List. </li>
-            <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-data"><strong>Data Files</strong></a>:  Machine readable files describing all of the licenses and license exceptions.</li>
-            <li style="margin:0px 0 5px 10px;"><a href="http://spdx.org/spdx-license-list/license-list-overview"><strong>Overview</strong></a>: General information about the SPDX License List, including principles for inclusion of a license and an explanation of the fields contained on the list.</li>
-            <li style="margin:0px 0 5px 10px;"><a href="http://spdx.org/ids"><strong>IDs</strong></a>: Learn more about using short-form license identifiers in your source code.</li>
-            <li style="margin:5px 0 5px 10px;"><a href="http://spdx.org/spdx-license-list/matching-guidelines"><strong>Matching Guidelines</strong></a>:  Guidelines for what constitutes a license match to the SPDX License List. The license text on the HTML pages here will display omitable text in blue and replaceable text in red (see Guideline #2 for more information).</li>
-            <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML/blob/master/CONTRIBUTING.md"><strong>Request New License</strong></a>: Instructions for proposing a license or exception be added to the SPDX License List.</li>
+            <li style="margin:5px 0 5px 10px;"><a href="./exceptions-index.html"><strong>License Exceptions</strong></a> are commonly found exceptions to free and open source licenses, used with the <strong><a href="https://spdx.org/specifications">License Expression</a></strong> operator, "WITH" to create a license with an exception. </li>
```

Comment:
The second URL (for the specifications link) can be updated to https://spdx.dev/specifications/